index.js

```diff
@@ -1,5 +1,127 @@
+/* eslint-disable consistent-return */
+/* eslint-env node */
+
 'use strict';
 
+const mergeTrees = require('broccoli-merge-trees');
+const Rollup = require('broccoli-rollup');
+const rollupConfig = require('./config/rollup');
+const outputConfig = require('./config/output');
+const path = require('path');
+
 module.exports = {
-  name: require('./package').name
+	name: require('./package').name,
+
+	isDevelopingAddon: () => true,
+
+	config(env, baseConfig) {
+		const options = baseConfig[this.name] || {};
+
+		env = env || process.env.EMBER_ENV;
+		const isProductionEnv = Boolean(env.match('prod'));
+		const url = baseConfig.rootURL || '';
+
+		const defaults = {
+			entrypointFileName: 'module-imports.js',
+			outputFileName: 'bundle.js',
+			outputPath: 'assets',
+			minify: isProductionEnv,
+			modules: false,
+			entrypointPaths: [],
+			autoImport: true
+		};
+
+		this.options = Object.assign(defaults, options);
+		this.options.rootURL = url;
+
+		this._setOutputOptions();
+	},
+
+	_setOutputOptions() {
+		this._outputFilePaths = this.options.entrypointPaths.map((dirname) =>
+			path.basename(dirname, this.options.entrypointFileName)
+		);
+
+		const config = outputConfig(this.app.project.targets);
+
+		this._outputConfigs = [config.legacy];
+
+		if (this.options.modules) {
+			this._outputConfigs.push(config.modules);
+		}
+	},
+
+	/* istanbul ignore next */
+	afterInstall() {
+		return this.addPackageToProject('@webcomponents/webcomponentsjs', '2.2.10');
+	},
+
+	included(app, parentAddon) {
+		this._super.included.apply(this, arguments);
+		this.app = parentAddon || app;
+
+		this._importPolyfills();
```

Comment:
This addon can be loaded in async (see `_getScriptTag`) so, the polyfills should be added to the `vendor.js` file instead of defer its loading?

Reply:
We can check if `autoImport` option is `true` to not include the polyfills in vendor, but this requires the user to know that `webcomponents-bundle.js` and `webcomponents-utils.js` (inside the addon) should be imported first... The second file is the problem here. Let me think about that.

Reply:
Polyfills are not included if `autoImport` option is set to `false`. 
Changed in a65a500